torch/cuda/_memory_viz.py

```diff
@@ -430,11 +463,11 @@ def profile_plot(memory_profile, device=None):
     def add_element(size, tensor_key, version):
         category = memory_profile._categories.get(tensor_key, version)
         if category is None:
-            category = 'unknown'
-        else:
-            category = category.name.lower()
+            category = Category.TEMPORARY
```

Comment:
Why the change? Just because something is uncategorized doesn't mean it's `TEMPORARY`.

Reply:
I didn't know what to categorize something without a category. Should we make an unknown or uncategorized category in the legend?